src/internals/StreamHttpReply.cpp

```diff
@@ -32,8 +32,11 @@ void ArduinoHttpServer::AbstractStreamHttpReply::send(const String& data, const
    httpErrorReply += title + "\r\n";
    httpErrorReply += AHS_F("Connection: close\r\n");
    httpErrorReply += AHS_F("Content-Length: ");
-   httpErrorReply += data.length(); httpErrorReply += AHS_F("\r\n");
-   httpErrorReply += AHS_F("Content-Type: "); httpErrorReply += m_contentType; httpErrorReply+= AHS_F("\r\n");
+   httpErrorReply += data.length();
```

Comment:
It is Oké, but initially I intended here to have one HTTP line per line of code, that's why I put the \r\n behind it.

Reply:
I dont mind either way but if you want one line for HTTP header than we need to fix the other ones as this is not currently the case. Let me know and I will change.

Reply:
Oké, just leave it, I will check and fix if needed afterwards.